source/codegen/metadata/nidaqmx/functions.py

```diff
@@ -125,6 +125,108 @@
         ],
         'returns': 'int32'
     },
+    'ReadDigitalU16': {
+        'parameters': [
+            {
+                'direction': 'in',
+                'name': 'task',
+                'type': 'TaskHandle',
+                'grpc_type': 'nidevice_grpc.Session'
+            },
+            {
+                'direction': 'in',
+                'name': 'numSampsPerChan',
+                'type': 'int32',
+            },
+            {
+                'direction': 'in',
+                'name': 'timeout',
+                'type': 'double',
+            },
+            {
+                'direction': 'in',
+                'name': 'fillMode',
+                'type': 'int32',
+            },
+            {
+                'direction': 'out',
+                'name': 'readArray',
+                'type': 'uInt16[]',
+                'grpc_type': 'repeated uint32',
+                'size': {
+                    'mechanism': 'passed-in',
+                    'value': 'arraySizeInSamps'
+                }
+            },
+            {
+                'direction': 'in',
+                'name': 'arraySizeInSamps',
+                'type': 'uInt32',
+                'grpc_type': 'uint32'
+            },
+            {
+                'direction': 'out',
+                'name': 'sampsPerChan',
+                'type': 'int32'
+            },
+            {
+                'direction': 'in',
+                'name': 'reserved',
+                'type': 'bool32*',
+                'grpc_type': 'uint64',
+            }
+        ],
+        'returns': 'int32',
+    },
+    'WriteDigitalU16': {
+        'parameters': [
+            {
+                'direction': 'in',
+                'name': 'task',
+                'type': 'TaskHandle',
+                'grpc_type': 'nidevice_grpc.Session'
+            },
+            {
+                'direction': 'in',
+                'name': 'numSampsPerChan',
+                'type': 'int32',
+            },
+            {
+                'direction': 'in',
+                'name': 'autoStart',
+                'type': 'int32',
+            },
+            {
+                'direction': 'in',
+                'name': 'timeout',
+                'type': 'double',
+            },
+            {
+                'direction': 'in',
+                'name': 'dataLayout',
+                'type': 'int32',
+            },
+            {
+                'direction': 'in',
+                'name': 'writeArray',
+                'type': 'const uInt16*',
```

Comment:
Can these be `const uInt16[]` instead of pointer just to be more consistent (both with the rest of grpc-device and with the daq header)?